webrtc.html

```diff
@@ -15505,7 +15503,6 @@ <h3>
                 {{RTCReceivedRtpStreamStats/packetsReceived}},
                 {{RTCReceivedRtpStreamStats/packetsLost}},
                 {{RTCReceivedRtpStreamStats/jitter}},
-                {{RTCReceivedRtpStreamStats/packetsDiscarded}},
```

Comment:
`packetsDiscarded` seems [well implemented](https://wpt.fyi/results/webrtc/RTCPeerConnection-mandatory-getStats.https.html?label=master&label=experimental&aligned).

Reply:
It's also [still in the stats spec](https://w3c.github.io/webrtc-stats/#dom-rtcinboundrtpstreamstats-packetsdiscarded) AFAICT

Reply:
It's implemented in inbound-rtp, not remote-inbound-rtp. The WPT tests both, one passes on all browsers and one fails on all browsers.

The PR moved it from RTCReceivedRtpStreamStats which implied both inbound-rtp and remote-inbound-rtp to only RTCInboundRtpStreamStats. This matches the implementation status

Reply:
Thanks for explaining! LGTM then